addOns/websocket/src/main/java/org/zaproxy/zap/extension/websocket/ui/WebSocketUiHelper.java

```diff
@@ -155,7 +155,7 @@ public JScrollPane getOpcodeMultipleSelect() {
 
     private JList<String> getOpcodeList() {
         if (opcodeList == null) {
-            int itemsCount = WebSocketMessage.OPCODES.length + 1;
+            int itemsCount = WebSocketMessage.getOpcodes().size();
```

Comment:
Why no longer +1 ? (Same for following statement.)

Reply:
Be sure I thought array length was off by one but size was accurate